projects/stargazer/plugins/other/radius/radius.cpp

```diff
@@ -109,9 +109,9 @@ int RAD_SETTINGS::ParseSettings(const ModuleSettings & s)
         auto pva = std::find(pvi->sections.begin(), pvi->sections.end(), pv);
         if (pva != pvi->sections.end() && !pva->value.empty())
         {
-            printfd(__FILE__, "ParseSettings Value of send: '%s'\n", pva->value[0].c_str());
+            printfd(__FILE__, "ParseSettings Value of auth/send: '%s'\n", pva->value[0].c_str());
 
-            m_sendPairs = ParseAuthAttr(pva->value[0], pv.param);
+            m_sendPairs = ParseAuthAutzAttr(pva->value[0], pv.param);
 
             for (const auto& at : m_sendPairs)
                 printfd(__FILE__, "Key: '%s', Value: '%s', Type: %d\n", at.first.c_str(), at.second.value.c_str(), at.second.type);
```

Comment:
Let's make a function, something like this:
```c++
std::string showRules(const std::vector<std::pair<std::string, AttrValue>>)
{
    // ...
}
```
Which will combine the pairs back to the original form, and use it like this:
```c++
printfd(__FILE__, "auth.match = \"%s\"\n", showRules(m_matchPairs).c_str());
...
```